doc/src/manual/variables-and-scoping.md

```diff
@@ -152,27 +150,18 @@ julia> z
 10
 ```
 
-Multiple global or local definitions can be on one line and can also be paired with assignments:
```

Comment:
It seems that it is only when paired with assignment that is deprecated:

```julia
julia> global a, b

julia> global x = 1; d

WARNING: deprecated syntax "global c=1, d" at REPL[2]:1.
Use "global c=1; global d" instead.
```

It seems a bit overkill to say that if you wanna pair it with assignment the keyword needs to be in front of each variable?

On another note: There is an empty line before the warning which is not the case for other warnings